src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/WindowsRuntime/Windows/Data/Text/WordsSegmenter.cs

```diff
@@ -93,6 +42,10 @@ private WordsSegmenter(string language)
             /// <exception cref="PlatformNotSupportedException">The OS platform is not supported</exception>
             public static WordsSegmenter Create(string language, bool shouldPreferNeutralSegmenter = false)
             {
+                if (Environment.OSVersion.Platform == PlatformID.Win32NT && Environment.OSVersion.Version < new Version(10, 0))
```

Comment:
See prev note about os checks

Reply:
We certainly support WordsSegmenter on OS < 10. This check is semantically wrong. It should be checked against win8.1 to prevent regressions.

Reply:
I'll change this to check against 8.1 instead of 10.